contracts/DataStructures/PriorityQueue.sol

```diff
@@ -0,0 +1,107 @@
+pragma solidity ^0.4.0;
+
+import 'zeppelin-solidity/contracts/math/SafeMath.sol';
+
+/**
+ * @title PriorityQueue
+ * @dev A priority queue implementation
+ */
+contract PriorityQueue {
```

Comment:
The path of this file is weird. It's using PascalCase instead of lower case for folder path name.
Perhaps it should be in `lib` if it's an external contract.